backend/apps/interest/models.py

```diff
@@ -0,0 +1,40 @@
+import uuid
+from django.db import models
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+
+class Interest(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+    investor = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="expressed_interests"
+    )
+
+    project = models.ForeignKey(
+        "projects.Project",
+        on_delete=models.CASCADE,
+        related_name="expressed_interests"
+    )
+
+    message = models.TextField(blank=True, null=True)
+
+    contact_opt_in = models.BooleanField(default=False)
+
+    status = models.CharField(
+        max_length=32,
+        default="expressed",
+        choices=[
+            ("expressed", "Expressed"),
+            ("contacted", "Contacted"),
+            ("ignored", "Ignored"),
+        ]
+    )
```

Comment:
it is better to put choices in Enum

    from django.db import models
    
    class InterestStatus(models.TextChoices):
        EXPRESSED = "expressed", "Expressed"
        CONTACTED = "contacted", "Contacted"
        IGNORED = "ignored", "Ignored"
    
    status = models.CharField(
        max_length=32,
        choices=InterestStatus.choices,
        default=InterestStatus.EXPRESSED,
    )

Advantages:
- fewer code errors
- autocomplete in IDE
- easier to modify/extend